src/solvers/elasticity_solver.cpp

```diff
@@ -42,14 +42,14 @@ ElasticitySolver::ElasticitySolver(int order, std::shared_ptr<mfem::ParMesh> pme
   displacement_->name = "displacement";
 }
 
-void ElasticitySolver::setDisplacementBCs(std::set<int>&                           disp_bdr,
+void ElasticitySolver::setDisplacementBCs(const std::set<int>&                     disp_bdr,
                                           std::shared_ptr<mfem::VectorCoefficient> disp_bdr_coef, int component)
 {
   setEssentialBCs(disp_bdr, disp_bdr_coef, *displacement_->space, component);
 }
 
-void ElasticitySolver::setTractionBCs(std::set<int>& trac_bdr, std::shared_ptr<mfem::VectorCoefficient> trac_bdr_coef,
-                                      int component)
+void ElasticitySolver::setTractionBCs(const std::set<int>&                     trac_bdr,
+                                      std::shared_ptr<mfem::VectorCoefficient> trac_bdr_coef, const int component)
```

Comment:
If we're passing POD by value, is it still a good idea to decorate with `const`?

Reply:
The [isocpp section on this](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#S-const) indicates that it's not necessary (and pedantic!), but I like to do it because I've seen bugs before where a function was changing an argument when it was supposed to be changing a function-local variable.